frontend/workflows/envoy/build.mjs

```diff
@@ -0,0 +1,32 @@
+import esbuild from "esbuild";
```

Comment:
Should we add this file to the workflow scaffolder in this pr?

Reply:
I'd like to figure out how to package this script in a way where it's easier to distribute.